response_operations_ui/controllers/sample_controllers.py

```diff
@@ -16,7 +16,7 @@ def upload_sample(short_name, period, file):
 
     survey_type = 'B'
     url = f'{app.config["SAMPLE_URL"]}/samples/{survey_type}/fileupload'
-    response = requests.post(url=url, auth=app.config['SAMPLE_AUTH'], files={'file': file})
+    response = requests.post(url=url, auth=app.config['SAMPLE_AUTH'], files={'file': (file.filename, file)})
```

Comment:
This isn't actually posting a file, it's posting a FileStorage object. You may want to alter the part where the FileStorage object is accessed and us it's `.read()` method to read it as binary data. This will play nicely with requests content-header settings.

From the requests docs:
`Warning
It is strongly recommended that you open files in binary mode. This is because Requests may attempt to provide the Content-Length header for you, and if it does this value will be set to the number of bytes in the file. Errors may occur if you open the file in text mode.`